activesupport/lib/active_support/callbacks.rb

```diff
@@ -826,6 +754,30 @@ def get_callbacks(name)
       def set_callbacks(name, callbacks)
         send "_#{name}_callbacks=", callbacks
       end
+
+      def default_false_terminator
+        Proc.new do |target, result_lambda|
+          terminate = true
+          catch(:abort) do
+            result = result_lambda.call if result_lambda.is_a?(Proc)
+            if CallbackChain.halt_and_display_warning_on_return_false && result == false
```

Comment:
It seems to me that there is not longer a configuration option `config.active_support.halt_callback_chains_on_return_false`, as you can see from 0a120a818d413c64ff9867125f0b03788fc306f8. Am I mistaken?

I suggest having a new PR that would also fix the docs that still mention the option. If not on `ActiveSupport`, would the configuration go to a `Callbacks` module?